.travis.yml

```diff
@@ -6,9 +6,6 @@ node_js:
   - "4.0"
   - "stable"
 
-before_install:
-  - npm install -g npm
-
```

Comment:
Please remove this commit. I'm not sure which older version of Node.js it was trying to support, but we probably don't support it anymore. Recently support for Node.js verisons < 4 was dropped

Reply:
Yep, it was breaking versions < 4 on travis